javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/ExtensionClassLoader.java

```diff
@@ -61,10 +69,20 @@ public static ClassLoader getInstance(
 
     includeEmbeddedExtensionsIfFound(extensions, javaagentFile);
 
-    extensions.addAll(parseLocation(earlyConfig.getString(EXTENSIONS_CONFIG), javaagentFile));
+    String externalExtensionsLocation = earlyConfig.getString(EXTENSIONS_CONFIG);
+    if (externalExtensionsLocation != null && !externalExtensionsLocation.isEmpty()) {
+      List<URL> externalExtensions = parseLocation(externalExtensionsLocation, javaagentFile);
+      if (externalExtensions.isEmpty()) {
```

Comment:
might be nice to log warning when there are multiple extensions, but one of them is invalid location?